packages/apps/human-app/frontend/src/i18n/en.json

```diff
@@ -17,15 +17,15 @@
   },
   "validation": {
     "required": "Required",
-    "min": "Min {{count}} characters",
+    "min": "Password should have at least eight characters.",
     "max": "Max {{count}} characters",
     "passwordWeak": "Password should be at least 8 characters long and contain at least one uppercase and one lowercase letter, one number and one special character",
     "passwordMismatch": "Passwords are not the same",
     "password8Chars": "8 characters",
     "passwordUppercase": "1 uppercase (A-Z)",
     "passwordLowercase": "1 lowercase (a-z)",
     "passwordNumeric": "1 numeric (0-9)",
-    "passwordSpecialCharacter": "1 special character (@#$%^&+=!*])",
+    "passwordSpecialCharacter": "1 special character (@#$%^&+=!*)",
```

Comment:
I think we don't need this since the only restriction we'll have for the password is length.

Reply:
done